README.md

```diff
@@ -73,6 +73,11 @@ You can read more about it [here](https://www.typescriptlang.org/tsconfig#esModu
 
 * Your function is now fully instrumented
 
+## Connect Your Lumigo Account
+Set your Lumigo token as the `LUMIGO_TRACER_TOKEN` environment variable of your Lambda function; refer to the [Using AWS Lambda environment variables](https://docs.aws.amazon.com/lambda/latest/dg/configuration-envvars.html) documentation for more information. Your Lumigo token is available in `Settings -> Tracing -> Manual tracing`, see the [Lumigo Tokens](https://docs.lumigo.io/docs/lumigo-tokens) documentation.
+
+We advise you to use the most secure available to you to store secrets such as your `LUMIGO_TRACER_TOKEN`; refer to AWS Lambda's [Securing environment variables](https://docs.aws.amazon.com/lambda/latest/dg/configuration-envvars.html#configuration-envvars-encryption) documentation for guidance on keeping the values of your Lambda environment variables secure.
```

Comment:
The grammar doesn't add up "...use the most secure available..."

Reply:
ha, missed this - we have it on the website as well. I'll change it here but how can I also change it on the website?

Reply:
Talk to @lir-lumigo, he can change the docs

Reply:
Updated after consulting with him. He will also update the website.